src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/LicenceService.kt

```diff
@@ -908,4 +910,25 @@ class LicenceService(
     }
     return originalLicence
   }
+
+  private fun getNextLicenceVersion(currentVersion: String, status: LicenceStatus): String {
+    val (majorVersion, minorVersion) = getVersionParts(currentVersion)
+    return when (status) {
+      VARIATION_IN_PROGRESS -> (majorVersion + 1).toString()
```

Comment:
so we don't have a `.0` on variations.. presumably this is intentional?

Reply:
yes that was what they wanted. I did think of keeping the .0 to make it simpler and could then just change how it's displayed in the frontend.

Reply:
Ah, the script that I've written to populate existing versions will still have `.0` on the end. I can change it to remove that if this is the approach we'd rather take!

Reply:
ok, as discussed I will update this, it will make this code simpler and if required we can just strip the .0 off variations in the front end.

Reply:
updated